packages/grafana-data/src/types/logs.ts

```diff
@@ -183,6 +183,9 @@ export interface DataSourceWithLogsContextSupport<TQuery extends DataQuery = Dat
     origQuery?: TQuery,
     scopedVars?: ScopedVars
   ): React.ReactNode;
+
+  // Does the datasource support the user adjusting the time range in the logs context window? https://github.com/grafana/grafana/pull/109901
+  hasLogsContextAdjustableWindow?: boolean;
```

Comment:
```suggestion
  supportsAdjustableWindow?: boolean;
```
It's already part of "logs context support", so maybe.